From dd2b384decc43ac07eca42ffcc216bd8f0e9e84f Mon Sep 17 00:00:00 2001
From: Richard Liu <r66033@freescale.com>
Date: Thu, 15 Aug 2013 14:21:56 +0800
Subject: [PATCH 234/310] ENGR00275387 set dpi to 120 on mx6sl to use tablet
 layout

mx6sl LCD panel resolution is 640x480 and dpi is 128,
android framework default dpi is set to 160 if you not
indicate (the 160 dpi and resolution 640x480 it will
go to phone layout), as 128 dpi is not a standard value,
here we set the dpi to 120 to align with google CCD.

Signed-off-by: Richard Liu <r66033@freescale.com>
---
 evk_6sl/init.rc | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/evk_6sl/init.rc b/evk_6sl/init.rc
index 3980402..0567335 100644
--- a/evk_6sl/init.rc
+++ b/evk_6sl/init.rc
@@ -52,8 +52,9 @@ on boot
     # Set rotation to 270 to cofigure as portrait mode
     setprop ro.sf.hwrotation 0
 
-    # Set the density to 160dpi, default 128dpi is not good
-    setprop ro.sf.lcd_density 160
+    # Set the density to 120dpi for 640x480 lcd panel
+    setprop ro.sf.lcd_density 120
+
 # change for FSL specific service
     chown root system /sys/devices/system/cpu/cpu0/cpufreq/scaling_max_freq
     chmod 0664 /sys/devices/system/cpu/cpu0/cpufreq/scaling_max_freq
-- 
1.8.0

